services/storage/src/simcore_service_storage/s3.py

```diff
@@ -32,7 +29,7 @@ async def _setup_s3_bucket(app):
         reraise=True,
     )
     async def do_create_bucket():
-        s3_cfg = cfg[_SERVICE_NAME]
+        s3_cfg = cfg["STORAGE_S3"]
```

Comment:
I'll comment on it here. But I've noticed it in the entire service after the refactor.

 Why are the settings accessed via keys and not via properties if they are backed up by a Pydantic model?